src/pudl/metadata/fields.py

```diff
@@ -3329,6 +3339,46 @@
         "type": "string",
         "description": "Name of the seller, or the other party in an exchange transaction.",
     },
+    "served_arbitrage_applications": {
+        "type": "boolean",
+        "description": "Whether the energy storage device served arbitrage applications during the reporting year",
+    },
+    "served_backup_power_applications": {
+        "type": "boolean",
+        "description": "Whether the energy storage device served backup power applications during the reporting year.",
+    },
+    "served_co_located_renewable_firming": {
+        "type": "boolean",
+        "description": "Whether the energy storage device served renewable firming applications during the reporting year.",
```

Comment:
Why does this boolean column not end in `_applications` while all the others do?

If they *all* end in `_applications` then is that suffix adding useful information to the column name, or could we drop it and have shorter names?

Reply:
hmm I think this one might have slipped through the cracks. The reason I added `application` as a suffix was mostly for readability / comprehension. The columns are kind of weird and I figured it makes more sense to have _verb, adjective, noun_ than just _verb, adjective_. The verb `served` isn't ideal for clarity, but I can't really think of a better one here.